packages/core/deploy/027_deploy_long_short_pair_creator.js

```diff
@@ -0,0 +1,20 @@
+const { ZERO_ADDRESS } = require("@uma/common");
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const Finder = await deployments.get("Finder");
+  const TokenFactory = await deployments.get("TokenFactory");
+  const Timer = await deployments.getOrNull("Timer");
+
+  await deploy("LongShortPairCreator", {
+    from: deployer,
+    args: [Finder.address, TokenFactory.address, Timer.address || ZERO_ADDRESS],
```

Comment:
Won't this break?

If `Timer` is null, won't we get an error when trying to access `null.address`?

```
> null.address
Uncaught TypeError: Cannot read property 'address' of null
```

If you look in some of the other files, we do it slightly differently to avoid this issue:
```js
const Timer = (await deployments.getOrNull("Timer")) || { address: ZERO_ADDRESS };

...
args: [Finder.address, TokenFactory.address, Timer.address],
...
```

Reply:
hhmm. the code example you put there would work fine as you pulling out the address. the code I had won't break per say but it is just undefined and then defaults to the `ZERO_ADDRESS`.

I have changed it to map to the implementation in the other scripts like you described.